$NetBSD: patch-ao,v 1.1 2005/12/29 20:10:23 ghen Exp $

SQL_NO_DATA is called SQL_NO_DATA_FOUND in ODBC versions >= 3.0,
as provided by databases/iodbc and databases/unixodbc.


--- servers/slapd/back-sql/delete.c.orig	2005-08-09 23:04:06.000000000 +0200
+++ servers/slapd/back-sql/delete.c
@@ -363,7 +363,7 @@ backsql_delete( Operation *op, SlapReply
 
 	rc = SQLExecute( sth );
 	switch ( rc ) {
-	case SQL_NO_DATA:
+	case SQL_NO_DATA_FOUND:
 		/* apparently there were no "auxiliary" objectClasses
 		 * for this entry... */
 	case SQL_SUCCESS:
